docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+### How will network retrieval of real data be measured and assessed on an incentivized testnet?
+
+During the competition, once your miner has > 0 storage power, many bots will begin attempting storage and retrieval deals with your miner. The competition dashboard will display your deal success rate in near-real time. Miners below a certain high threshold will be ineligible to receive rewards.
+
+We plan to run the bots for one to two weeks before the competition to give miners a chance to test their configuration, and will announce when that begins.
+
+### What do miners have to do to change a committed capacity (CC) sector into a "real-data" sector?
+
+We are reusing much of the existing code path for this first iteration. Miners will publish storage deals that they will upgrade the CC sector with, announce to the chain that they are doing an upgrade, and prove to the chain that a new sector has been sealed correctly. We expect to evolve and make this cheaper and more attractive over time after mainnet.
+
+### For the incentive phase, you said a miner has to execute the full sector life cycle and terminate the sectors. What does “terminating” the sectors mean?
+
+When a committed capacity sector is added to the chain, it can upgrade to a sector with deals, extend its lifetime, or terminate through either faults or voluntary actions. While we don’t expect this to happen very often on mainnet, a miner may deem it rational to terminate their promise to the network and their clients but accept a penalty for doing so. We want miners to execute the full sector lifecycle to get a flavor of providing long-term and useful storage on Filecoin.
+
+### Does the committed capacity sector still need to be sealed before it upgrades to one with real data?
+
+For the first iteration of the protocol, yes. We have plans to make it cheaper and more economically attractive after mainnet with no resealing required and other perks.
+
+### What’s the minimum time period for the storage contract between the provider and the buyer?
+
+The minimum duration for a deal is set in the miner’s ask. There’s also a practical limitation, because sectors have a minimum duration (currently one month).
+
+### After I made a deal with a miner, and sent my data to them, how exactly is the data supposed to be recoverable and healable if that miner goes down?
+
+Automatic repair of faulted data is a feature we've pushed off until after mainnet launch. For now, the way to ensure resiliency is to store your data with multiple miners, to gain some level of redundancy. If you want to learn more about how we are thinking about repair in the future, [here are some notes](https://github.com/filecoin-project/specs/pull/245/files).
+
+### How do I know that whatever miner "takes the burden" would not immediately start asking me ridiculously high prices for retrieval?
```

Comment:
How do I know that my storage miner will not charge prohibitively high costs for data retrieval?